Solutions/Fortinet-FortiGate/Playbooks/Fortinet_ResponseOnURL/azuredeploy.json

```diff
@@ -43,12 +67,12 @@
     }
   },
   "variables": {
-    "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('Playbook Name'))]",
-    "TeamsConnectionName": "[concat('teamsconnector-', parameters('Playbook Name'))]",
-    "FortinetConnectorConnectionName": "[concat('FortinetConnector-', parameters('Playbook Name'))]",
+    "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]",
+    "TeamsConnectionName": "[concat('teamsconnector-', parameters('PlaybookName'))]",
+    "FortinetConnectorConnectionName": "[concat('FortinetConnector-', parameters('PlaybookName'))]",
     "UserIdentifier": "[parameters('User Identifier Name')]",
-    "functionappname": "[parameters('Function App Name')]"
-
+    "functionappname": "[parameters('Function App Name')]",
+    "AzureManagementLink": "[concat('https://management','.azure','.com')]"
```

Comment:
@v-amolpatil read the FortinetConnector name from parameter in this file as well